app/swagger/schemas/gibct/institution_programs.rb

```diff
@@ -37,20 +37,34 @@ class InstitutionPrograms
               property :attributes do
                 key :required, %i[facility_code description]
 
-                property :program_type, type: %i[null string],
-                                        enum: %w[IHL NCD OJT FLGT CORR]
-                property :description, type: :string, example: 'COMPUTER SCIENCE'
-                property :length_in_hours, type: %i[null string], example: '680'
-                property :length_in_weeks, type: %i[null integer], example: 12
-                property :facility_code, type: :string, example: '1X12345'
-                property :institution_name, type: :string, example: 'CODE PLACE'
-                property :city, type: %i[null string], example: 'ANYTOWN'
-                property :state, type: %i[null string], example: 'WA'
-                property :country, type: %i[null string], example: 'USA'
-                property :preferred_provider, type: %i[null boolean], example: false
-                property :tuition_amount, type: %i[null integer], example: 1000
-                property :va_bah, type: %i[null number], example: 220
-                property :dod_bah, type: %i[null integer], example: 200
+                property :program_type, type: :string, example: 'NCD',
+                                        description: 'The classification of the program, ex: IHL, NCD, OJT'
+                property :description, type: :string, example: 'Computer Science',
+                                       description: 'Program name'
+                property :length_in_hours, type: :string, example: '680',
+                                           description: 'Length of program (in hours)'
+                property :length_in_weeks, type: :integer, example: 12,
```

Comment:
Confirmed this is strange, but accurate.  `length_in_hours` is a `string`, but `length_in_weeks` is an `integer` in the payload. 😕

Reply:
length_in_hours is completely worthless, because the data is not consistent in any way, hence unusable. The length_in_weeks is providing a consistent integer we can use.